packages/salesforcedx-vscode-replay-debugger/src/core/logContext.ts

```diff
@@ -88,6 +88,16 @@ export class LogContext {
     );
   }
 
+  public meetsLogLevelRequirements(): boolean {
+    return (
+      this.logLines &&
+      this.logLines.length > 0 &&
+      this.logLines[0].match(
+        /\d{2}.*APEX_CODE,FINEST;.*VISUALFORCE,FINEST;.*/
```

Comment:
Just a quick comment: this regex is good, since it accounts for the fact that some other log categories might come in later.